modules/code-editor/src/views/full/SidePanel.tsx

```diff
@@ -48,7 +48,9 @@ class PanelContent extends React.Component<Props> {
     const files = this.props.files[fileType]
 
     if (files && files.length) {
-      fileList.push({ label, files })
+      const sorted_files = files.slice().sort((a, b) => a.location.localeCompare(b.location))
```

Comment:
I doubt `.slice()` is necessary here

Reply:
```suggestion
      const sortedFiles = files.slice().sort((a, b) => a.location.localeCompare(b.location))
```

Reply:
Or even better: `_.sortBy(files, 'location')`